docs/reference/mapping.asciidoc

```diff
@@ -5,14 +5,39 @@
 --
 
 Mapping is the process of defining how a document, and the fields it contains,
-are stored and indexed. For instance, use mappings to define:
-
-* which string fields should be treated as full text fields.
-* which fields contain numbers, dates, or geolocations.
-* the <<mapping-date-format,format>> of date values.
-* custom rules to control the mapping for
+are stored and indexed. You can choose how and where to map your data. Each
+method provides different benefits based on where you are in your data journey:
+
+* <<dynamic-field-mapping,Dynamic field mappings>> allow you to experiment with
+and explore data when you’re just getting started. {es} will add new fields
+automatically, just by indexing a document. You can add fields to the top-level
+mapping, and to inner <<object,`object`>>  and <<nested,`nested`>> fields.
+* <<runtime-search-request,Runtime fields in search requests>> are flexible,
+allowing you to experiment with different mapping options.
+* <<runtime-mapping-fields,Runtime fields in index mappings>> allow you to make
+schema changes without reindexing. You can use these fields in conjunction with
+indexed fields to balance resource usage and performance. Your index will be
+smaller, but with slower search performance.
+* <<explicit-mapping,Explicit mappings>> provide the best search performance
+but require more storage space, resulting in a larger index.
+
+When mapping your data using runtime fields or through explicit mapping, you
```

Comment:
what;s the difference between explicit mapping and using runtime fields?

Reply:
Hmm...other than using runtime fields in a search request, I guess that you're explicitly mapping fields on both accounts. That is, you're adding something to the mapping. The difference is that values for one are always stored in `_source`, while runtime fields can calculate values at query time. Correct?

Reply:
yes, to me mappings-wise, there is no difference. only runtime fields are defined in a separate section, but it is still under mappings. So after all you can explicitly map runtime fields or concrete fields, or you can automatically map runtime fields or concrete fields.